src/SDKs/Cdn/Management.Cdn/Generated/CdnManagementClient.cs

```diff
@@ -397,6 +397,8 @@ private void Initialize()
             DeserializationSettings.Converters.Add(new PolymorphicDeserializeJsonConverter<DeliveryRuleAction>("name"));
             SerializationSettings.Converters.Add(new PolymorphicSerializeJsonConverter<DeliveryRuleCondition>("name"));
             DeserializationSettings.Converters.Add(new PolymorphicDeserializeJsonConverter<DeliveryRuleCondition>("name"));
+            SerializationSettings.Converters.Add(new PolymorphicSerializeJsonConverter<CustomDomainHttpsParameters>("certificateSource"));
```

Comment:
@nachakra did you add this after the generated code, if yes, all these custom settings should be added inside CustomInitialize() that you will be defining in your customization layer.
Create a customization folder and add a new class file with the same exact class name (as all classes partial)
Declare CustomInitialize() in it and add whatever custom code you need.

Reply:
I did not change anything in the generated code. Just refactored the tests.